keps/sig-node/4815-dra-partitionable-devices/kep.yaml

```diff
@@ -2,6 +2,7 @@ title: DRA Partitionable Devices
 kep-number: 4815
 authors:
   - "@klueska"
+  - "@mortent"
```

Comment:
Are we actually changing any sig-node code? Or all scheduling code? Should "owning" sig be scheduling and "participating" be node?

cc @mrunalp @alculquicondor @dom4ha @dchen1107 @klueska @pohly @Huang-Wei

Reply:
That is a good question. I don't think there is anything in this KEP that requires changing sig-node code, although unlike the prioritized-list feature, this seems a bit "closer" to sig-node since it impacts the `ResourceSlice` API, which is mostly used by the drivers.

Might be better to have this discussion on https://github.com/kubernetes/enhancements/pull/5066, as it is another change to the same KEP, but with a smaller scope.

Reply:
sig-scheduling sounds appropriate

Reply:
Sounds good. Unless someone objects, I'll update https://github.com/kubernetes/enhancements/pull/5066 to move this KEP to sig-scheduling and once that is merged, I'll update this one.

Reply:
Sounds good!

Reply:
https://github.com/kubernetes/enhancements/pull/5066 moved the KEP to sig-scheduling.